medicaltorch/transforms.py

```diff
@@ -46,21 +48,25 @@ def __call__(self, sample):
         rdict = {}
         input_data = sample['input']
 
-        if isinstance(input_data, list):
-            ret_input = [F.to_tensor(item)
-                         for item in input_data]
+        if len(input_data) > 1:
+            # Multiple inputs
+            ret_input = [F.to_tensor(item) for item in input_data]
         else:
-            ret_input = F.to_tensor(input_data)
+            # single input
+            ret_input = F.to_tensor(input_data[0])
+            # transform list of dic into single dic
+            rdict['input_metadata'] = sample['input_metadata'][0]
```

Comment:
What about input_metadata for len>1?

Reply:
input_metadata is a _list_ of _dict_ by default (c.f. [L352](https://github.com/neuropoly/medicaltorch/blob/ab/generalization_dataset/medicaltorch/datasets.py#L352)). In case of a single input, there is just one _dict_.